### DISCUSSION TITLE
Different panels on different desktops or monitors?

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#2212] [Ideas]
This came to my mind. It would allow creating special "panel toolbars" for special tasks.

**Comment by tsujan** [comment]
Headache for coders?

**Reply by stefonarch** [reply]
Probably :) Not sure it it's a good idea too... having some quicklauncher for graphic tools and for example colorpicker ecc only on one specific desktop.

**Reply by tsujan** [reply]
Per-desktop panels are theoretically possible but, since different WMs can have totally different behaviors, unnecessary problems will arise. It's too fancy, IMO; it might not be if we had a WM of our own.

I was talking about per-screen panels. The idea was suggested somewhere before. Having corrected Panel's behavior with multiple screens in several commits, I prefer to stay away from future hassles that it might cause :P

**Reply by tsujan** [reply]
By supporting per-screen panels I mean restoring a panel of a removed screen when the screen is added again. Otherwise, the user can already have multiple panels on multiple screens as far as no part of them is between two screens.
